### DISCUSSION TITLE
Testing

### DISCUSSION THREAD

**Discussion by JariHanah** [community/community#88546] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Is there where I ask a question to the author of the code?

**Comment by subidit** [comment]
Check the README of the repo to find if the author has mentioned any preferred communication method, if not then you could either use the discussion tab of the repo (if it's there) or you could raise an issue.

**Comment by moheedinamdar** [comment]
You can open an issue on the project's GitHub repository.
To open an issue, go to the repository, click on the "Issues" tab, and then click the "New Issue" button.
OR
You can also comment on specific lines of code within a file. 
Navigate to the file, click on the line number, and a comment box will appear. This allows you to ask questions or provide feedback about a particular part of the code.
